src/test/bundles/07-helm-overrides/values-file/uds-bundle.yaml

```diff
@@ -0,0 +1,43 @@
+kind: UDSBundle
+metadata:
+  name: helm-values-file
+  description: testing a bundle with Helm overrides
+  version: 0.0.1
+
+packages:
+  - name: helm-overrides
+    path: "../../../packages/helm"
+    ref: 0.0.1
+
+    overrides:
+      podinfo-component:
+        unicorn-podinfo:
+          valuesFiles:
+            - values.yaml
```

Comment:
Can we also test what it looks to have multiple `valuesFiles`? Maybe just an add-on to the e2e test?